src/apis/jobTitle.js

```diff
@@ -2,16 +2,34 @@ import R from 'ramda';
 import graphqlClient from 'utils/graphqlClient';
 import {
   getJobTitleQuery,
+  queryJobTitleOverviewGql,
   getJobTitleTimeAndSalaryQuery,
   queryJobTitlesHavingDataGql,
 } from 'graphql/jobTitle';
 
+// TODO: DEPRECATED
 export const getJobTitle = jobTitle =>
   graphqlClient({
     query: getJobTitleQuery,
     variables: { jobTitle },
   }).then(R.prop('job_title'));
 
+export const queryJobTitleOverviewApi = ({
```

Comment:
想確定我們有把 API 當詞綴的計畫嗎？
個人覺得Api不用出現在 method 名稱裡面

```suggestion
export const queryJobTitleOverview = ({
```

Reply:
主要是因為 action, apis 都有叫做 queryJobTitleOverview，而他們的意涵有一點點差異，所以在想說是不是做點區隔

Reply:
要差異的話，我覺得可以在 import as 的時候改名字，只有在import的時候會和其他scope的同名者打架，但光在api的scope底下他本身不需要加詞綴

另外用Api當詞綴感覺也不算常見的practice（不像 selector詞綴比較常見..嗎？）

但這只是個人意見，不一定要同意，可以直接合沒關係。

Reply:
我 follow 你的建議～我的目標是全部的用法都一致就好～～